public/lessons/www/hello_world.md

```diff
@@ -42,4 +36,15 @@ Hello World!
  
 # Let's Open Our First Web Page!
 
-* We will now open our first web page using a web browser :)
+* We will now open our first web page using a web browser
+* Enter ```open index.html``` into your terminal
```

Comment:
inline code needs one backtick; fenced multi-line code blocks need three backticks... please fix this to be inline